### PR TITLE
SITE-6922 CalendarRefresh icon added for recurring campaign callout

### PR DESCRIPTION
**Pull request by msied**
Needed for https://crowdtilt.atlassian.net/browse/SITE-6922

### CODE REVIEW

**Comment by msied** [comment]
ping

File: app/stylesheets/CalendarRefresh.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@import 'support/colors';
```
Comment (tildedave) [inline app/stylesheets/CalendarRefresh.scss]: don't listen to this advice

**Comment by amackera** [comment]
yo, lgtm but I'm not a `tilt-images` master by any means

**Comment by msied** [comment]
ping

**Comment by kevzettler** [comment]
lgtm

**Comment by msied** [comment]
Ping, added two more that are relevant to the ticket

**Comment by amackera** [comment]
lgtm

**Comment by kevzettler** [comment]
lgtm

**Pull request merged**